config/enabled.yml

```diff
@@ -1478,3 +1478,10 @@ Rails/UniqBeforePluck:
 Rails/Validation:
   Description: 'Use validates :attribute, hash of validations.'
   Enabled: true
+
+Rails/KeyWordArgs:
```

Comment:
Btw, I think the name of this cop is quite confusing and should probably be something else - related to the methods it actually checks.